.gitignore

```diff
@@ -13,3 +13,4 @@
 /yarn-error.log
 /.happypack
 /junit.xml
+/lib
```

Comment:
This folder is not used anymore in this branch. That's why I removed it from the `.gitignore`. 😉